app/src/main/java/com/swayy/musicpark/data/local/dao/BaseDao.kt

```diff
@@ -0,0 +1,27 @@
+package com.swayy.musicpark.data.local.dao
```

Comment:
This is great to introduce a base dao for inheritance and reusability in the project reducing boilerplate code 👏👏 ata mimi sikuwa najua a person can do it this way learning never ends 😂

Reply:
student for life 😂